docs/docs/spark-configuration.md

```diff
@@ -154,6 +154,10 @@ spark.read
     .table("catalog.db.table")
 ```
 
+Iceberg 1.8.0 and later support setting read options by Spark session configuration `spark.datasource.iceberg.<key>=<value>`
```

Comment:
I still think we need new section like 'Configuration Priority' where we can explain the order of precedence:
Writes:
- explicit dataframeWriter option
- dataframe session default
- if table exists, explicit table option
- if table exists, table default

Reads:
- explicit dataFrameReader option
- dataframe session default
- if table exists, explicit table option
- if table exists, table default

Reply:
I hesitate to write such a section because the situation looks more complex, some configurations are allowed to be set by dedicated session configuration, for example

```
  public boolean localityEnabled() {
    boolean defaultValue = Util.mayHaveBlockLocations(table.io(), table.location());
    return confParser
        .booleanConf()
        .option(SparkReadOptions.LOCALITY)
        .sessionConf(SparkSQLProperties.LOCALITY)
        .defaultValue(defaultValue)
        .parse();
  }
```